tests/acceptance/test_organization_switch.py

```diff
@@ -28,15 +29,20 @@ def setUp(self):
 
         self.login_as(self.user)
 
-    @pytest.mark.skip(reason="Unstable right now.")
     def test_organization_switches(self):
-        def navigate_to_issues_page_and_select_projects(org_slug):
+        def navigate_to_issues_page(org_slug):
             issues_url = OrganizationSwitchTest.url_creator("issues", org_slug)
             self.browser.get(issues_url)
             self.browser.wait_until_not(".loading-indicator")
+
+        @TimedRetryPolicy.wrap(timeout=120, exceptions=(TimeoutException,))
```

Comment:
120 seconds is a bit too high of a timeout, with retries it will add 10 minutes if the tests always fail.

Reply:
I had forgotten that pytest has another layer of retries. I'll lower this so that the 5 attempts don't take so long.